docs/en/contributor/Contributor-Tools.md

```diff
@@ -275,7 +316,7 @@ RPCs on clients and implementing the RPCs on servers. To regenerate Java code af
 a gRPC definition, you must rebuild `alluxio-core-transport` module with `'generate'` maven profile.
 
 ```console
-$ mvn clean install -Pgenerate
+$ mvn clean install -Pgenerate -pl "org.alluxio:alluxio-core-transport"
```

Comment:
Building only the transport module saves a lot of time.

Reply:
👍